timestamp-impl/src/test/java/com/palantir/timestamp/PersistentTimestampServiceTest.java

```diff
@@ -90,7 +91,7 @@ public void incrementUpperLimitIfOneMinuteElapsedSinceLastUpdate() throws Interr
         Thread.sleep(10);
         persistentTimestampService.getFreshTimestamp();
         Thread.sleep(10);
-        verify(timestampBoundStore, times(2)).storeUpperLimit(anyLong());
+        verify(timestampBoundStore, atLeast(2)).storeUpperLimit(anyLong());
```

Comment:
Theoretically yes, but in practice I've never seen it happen -- for this test, for example, most of the time it's _exactly 2_, and occasionally 3. Even repeating this test hundreds of times, I haven't seen it be more than 3 calls to storeUpperLimit.